commands/issue.go

```diff
@@ -192,6 +202,7 @@ func listIssues(cmd *Command, args *Args) {
 			"creator":   flagIssueCreator,
 			"mentioned": flagIssueMentioned,
 			"labels":    flagIssueLabelsFilter,
+			"sort":      flagIssueSort,
```

Comment:
It doesn't: line 209 I check if the flag was passed before adding the sort parameter. But I'm going to add the check that sort and direction params are nil.